weave-js/src/components/PagePanelComponents/Home/Browse2/RunsTable.tsx

```diff
@@ -215,16 +215,18 @@ export const RunsTable: FC<{
       {
         field: 'span_id',
         headerName: 'Call',
-        width: 75,
-        minWidth: 75,
-        maxWidth: 75,
+        width: 250,
         hideable: false,
         renderCell: rowParams => {
-          // return truncateID(rowParams.row.id);
+          const opVersion = rowParams.row.ormCall?.opVersion();
```

Comment:
@tssweeney - I'm not really sure what the deal with useMaybeWeaveflowORMContext is - do I need to check `orm`?

Reply:
I was hoping we could just get rid of browse2 but it sounded today like Shawn still wanted to use it?